adafruit_hid/us_keyboard_layout.py

```diff
@@ -0,0 +1,218 @@
+# The MIT License (MIT)
```

Comment:
I'd rename this file keyboard_layout_us so that other layouts are sorted next to it. We could also make a keyboard_layout subpackage but that may be overkill.